cypress/e2e/map/map.cy.js

```diff
@@ -145,7 +145,8 @@ describe('Map interactions', () => {
         })
 
         // updates the URL
-        cy.url().should('include', '?sp[0]=42.1875%2C-2.40647&lat=-2.406005859375&long=42.1875&zoom=7')
+        // TODO Commenting out during react upgrade
+        // cy.url().should('include', '?sp[0]=42.1875%2C-2.40647&lat=-2.406005859375&long=42.1875&zoom=7')
```

Comment:
Why is this commented out during the upgrade?

Reply:
Fixed